modules/selenium/src/main/java/org/testcontainers/containers/SeleniumUtils.java

```diff
@@ -65,7 +65,7 @@ public static String determineClasspathSeleniumVersion() {
 
     /**
      * Read Manifest to get Selenium Version.
-     * @param manifest
+     * @param manifest manifest
```

Comment:
```suggestion
     * @param manifest
```

Let's just remove this, to reduce the changeset.

Reply:
@kiview Selenium 3+ still supports it but eventually I think they will drop it. Selenium already logs a warning that usage of ChromeOptions and FirefoxOptions is preferred and one of the Selenium devs also confirmed this.

Reply:
> Let's just remove this, to reduce the changeset.
If I remove it javadoc check will start complaining

Reply:
Okay cool, let's leave it like this then.